ci/test-large-network.sh

```diff
@@ -16,5 +16,7 @@ export LD_LIBRARY_PATH+=:$PWD
 
 export RUST_LOG=multinode=info
 
+ulimit -n 65000
```

Comment:
Can you document this number? What problem does it solve? Why might we want to change it?

Reply:
I'll add documentation.